webrender/src/picture.rs

```diff
@@ -467,6 +467,14 @@ impl PicturePrimitive {
                 pic_state.tasks.push(render_task_id);
                 self.surface = Some(PictureSurface::RenderTask(render_task_id));
 
+                // If the local rect of the contents changed, force the cache handle
+                // to be invalidated so that the primitive data below will get
+                // uploaded to the GPU this frame. This can occur during property
+                // animation.
+                if pic_state.local_rect_changed {
```

Comment:
when is it ever reset to `false`?

Reply:
The ```PictureState``` struct is constructed every frame, during the recursive traversal.

Reply:
(It's just a stack allocated helper struct passed down to child pictures).